### DISCUSSION TITLE
Script problem WARNUNG:  exited with status code 1603

### DISCUSSION THREAD

**Discussion by michaelortnersanube** [amidaware/tacticalrmm#1288] [Q&A]
Hello everyone,

i have a problem to install a software (3CX Client Version 18) the problem is, that winget and Chocolatey doesn't support this to install. So i have write a script what should make this. But if i install it to a client, following error show up in tactical rmm and the software will not install.

WARNUNG:  exited with status code 1603

My script:
```
Invoke-RestMethod "https://company.my3cx.at:5001/webclient/api/app/windows" -OutFile ( New-Item -Path "c:\temp\3CXDesktopApp-18.10.461.msi" -Force )
$proc = Start-Process c:\temp\3CXDesktopApp-18.10.461.msi -ArgumentList '-quiet' -PassThru

Wait-Process -InputObject $proc

if ($proc.ExitCode -ne 0) {
    Write-Warning "$_ exited with status code $($proc.ExitCode)"
}
Remove-Item -Path "c:\temp\3CXDesktopApp-18.10.461.msi" -Force
```

If i try this script with "Windows PowerShell ISE" it installs the software, but not with tactical rmm. Is it maybe because of the timeout?

**Comment by dinger1986** [comment]
It could be the timeout, I think your arguments are maybe wrong for the MSI, try the script in remote background and powershell from there.

Or use send a command and try the main parts one by one

**Reply by michaelortnersanube** [reply]
Okay, i have try it with background and powershell - here i have use following script to test this:

`Invoke-RestMethod "https://company.my3cx.at:5001/webclient/api/app/windows" -OutFile ( New-Item -Path "c:\temp\3CXDesktopApp-18.10.461.msi" -Force )
Start-Process c:\temp\3CXDesktopApp-18.10.461.msi -ArgumentList '-quiet' -PassThru`

and it has installed. But how i can make this on the script manager i think it is a problem of the timeout in tactical rmm

**Reply by dinger1986** [reply]
In the bottom left of the script manager when you create the script you can change the timeout, how long does it take to run? Make sure you increase it high enough.
